index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<script src="https://kit.fontawesome.com/yourcode.js" crossorigin="anonymous"></script>
+	<title>Portfolio</title>
+</head>
+<body>
+	<section class="screen_container">
```

Comment:
- [ ] Kindly consider removing this section tag as it is against semantic principles because our web page's body divides generally in two things: `header and main` tags. then the main tag is divided into `sections`. removing it will make your code semantically correct.

Reply:
Fixed.